### PR TITLE
docs: update changelog for 21.05.4

### PR DESCRIPTION
**Pull request by cgrinds**

### CODE REVIEW

**Review by ruanruijuan — COMMENTED** [review]
Looks like #303 is missed.

Should we add #284 in addition to #286 (they are dups)?

**Comment by cgrinds** [comment]
> Looks like #303 is missed.
> 
> Should we add #284 in addition to #286 (they are dups)?

Good catch. Added #303.

#284 is the issue and #286 is the pull request that fixed #284.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,55 @@
 
 [Releases](https://github.com/NetApp/harvest/releases)
```
Comment (rahulguptajss) [inline CHANGELOG.md:3]: missing 
1692df1 feat: supporting new_status metric in native dashboards
29fb793 refactor: use table for harvest status

Comment (cgrinds) [reply CHANGELOG.md:3]: first one is captured in 
> - New plugin [value_to_map](https://github.com/NetApp/harvest/blob/main/cmd/poller/plugin/README.md#value_to_num) helps map labels to numeric values for Grafana dashboards. Current dashboards updated to use this plugin [#319](https://github.com/NetApp/harvest/pull/319)

second not worth mentioning

**Review by rahulguptajss — DISMISSED** [review]

**Review by Hardikl — APPROVED** [review]

**Review by ruanruijuan — DISMISSED** [review]

**Review by vgratian — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,57 @@
...
+
+- Harvest Grafana tool (optionally) adds a user-specified prefix to all Dashboard metrics during import. See  `harvest grafana --help` [#87](https://github.com/NetApp/harvest/issues/87)
+  
+- Harvest is taking its first steps to talk REST: query ONTAP, show Swagger API, model, and definitions [#292](https://github.com/NetApp/harvest/pull/292)
+ 
+- Tagged releases of Harvest are published to [Docker Hub](https://hub.docker.com/r/rahulguptajss/harvest) 
+  
+- Harvest honors Go's http(s) environment variable proxy information. See https://pkg.go.dev/net/http#ProxyFromEnvironment for details [#252](https://github.com/NetApp/harvest/pull/252)
+
+- New plugin [value_to_map](https://github.com/NetApp/harvest/blob/main/cmd/poller/plugin/README.md#value_to_num) helps map labels to numeric values for Grafana dashboards. Current dashboards updated to use this plugin [#319](https://github.com/NetApp/harvest/pull/319)
```
Comment (Hardikl) [inline CHANGELOG.md:28]: value_to_num

**Pull request merged**